README.md

```diff
@@ -5,17 +5,14 @@
 
 <div align="center">
 
-
-
 <p>
-	<a href="https://github.com/mindsdb/mindsdb/actions"><img src="https://github.com/mindsdb/mindsdb/actions/workflows/release.yml/badge.svg" alt="MindsDB Release"></a>
+	<a href="https://github.com/mindsdb/mindsdb/releases"><alt="MindsDB Release"></a>
```

Comment:
This will not work since we need an image to be displayd

Reply:
Do we have any image? Or shall I remove it?

Reply:
Release is the same as a PyPI version so we may change the PyPI btn to be releases and remove this one